src/phasespace/load_mocap/load_mocap.py

```diff
@@ -219,14 +333,54 @@ def _reader_thread(self):
                         # print("%d: %f %f %f" % (m.id, m.x, m.y, m.z))
                 timestamp = np.array(time.time())
                 self._read_buffer.put((new_frame, timestamp))
+
```

Comment:
Hmm, good question. I kinda think we should just get rid of the buffer here, since as you say, the data is buffered in the individual iterators anyway. Let's chat briefly about this tomorrow to decide.